app/src/main/java/org/mozilla/fenix/IntentReceiverActivity.kt

```diff
@@ -63,6 +63,7 @@ class IntentReceiverActivity : Activity() {
     private fun setIntentActivity(intent: Intent) {
         val openToBrowser = when {
             components.utils.customTabIntentProcessor.matches(intent) -> {
+                // TODO This is potentially unsafe. Any app on the system can set the auth extra.
```

Comment:
Do we want to ship it, as is it?

Reply:
In case of true, should we file an issue for this pending work?

Reply:
Filed https://github.com/mozilla-mobile/fenix/issues/5225

Reply:
👍